resources/locale.json

```diff
@@ -745,195 +745,195 @@
+        "UI_PEACOCK_BLUEBERRYBUSH_OBJ3": "Eliminiere Aldo Trentino durch einen Unfall",
         "UI_PEACOCK_GLOBAL_CAMERA_FAIL": "Kamerascheu",
         "UI_PEACOCK_GLOBAL_CAMERA_FAIL_DESC": "Du scheiterst wenn du von einer Kamera entdeckt wirst.",
         "UI_PEACOCK_GRAPEBUSH": "Die Argentine Bitterkeit",
         "UI_PEACOCK_GRAPEBUSH1": "Die Argentine Bitterkeit - Stufe 1",
         "UI_PEACOCK_GRAPEBUSH2": "Die Argentine Bitterkeit - Stufe 2",
         "UI_PEACOCK_GRAPEBUSH3": "Die Argentine Bitterkeit - Stufe 3",
-        "UI_PEACOCK_GRAPEBUSH_FILES": "Get the Leak Report from the Basement Safe",
-        "UI_PEACOCK_GRAPEBUSH_FILES_DESC": "Get the Leak Report from the Basement Safe.",
-        "UI_PEACOCK_GRAPEBUSH_OBJ1": "Eliminate Violetta Mendez in an accident",
-        "UI_PEACOCK_GRAPEBUSH_OBJ2": "Eliminate Montel Beneventi in an explosion or with a cannon",
-        "UI_PEACOCK_GRAPEBUSH_OBJ2_DESC": "Eliminate Montel Beneventi in an explosion or with a cannon.",
-        "UI_PEACOCK_GRAPEBUSH_OBJ3": "[Optional] Eliminate Hira Amala",
-        "UI_PEACOCK_GRAPEBUSH_OBJ3_DESC": "If you're up for it, eliminate Hira Amala.",
+        "UI_PEACOCK_GRAPEBUSH_FILES": "Besorge den Leckbericht aus dem Kellertresor",
+        "UI_PEACOCK_GRAPEBUSH_FILES_DESC": "Besorge den Leckbericht aus dem Kellertresor.",
+        "UI_PEACOCK_GRAPEBUSH_OBJ1": "Eliminiere Violetta Mandez durch einen Unfall",
+        "UI_PEACOCK_GRAPEBUSH_OBJ2": "Eliminiere Montel Beneventi in einer Explosion oder mit einer Kanone",
+        "UI_PEACOCK_GRAPEBUSH_OBJ2_DESC": "Eliminiere Montel Beneventi in einer Explosion oder mit einer Kanone.",
+        "UI_PEACOCK_GRAPEBUSH_OBJ3": "[Optional] Eliminiere Hira Amala",
+        "UI_PEACOCK_GRAPEBUSH_OBJ3_DESC": "Wenn du dir das zutraust, eliminiere Hira Amala",
         "UI_LOCATION_HOKKAIDO_SHIM_MAMUSHI_CITY": "Hokkaido",
         "UI_LOCATION_HOKKAIDO_SHIM_MAMUSHI_COUNTRY": "Japan",
         "UI_LOCATION_HOKKAIDO_SHIM_MAMUSHI_TITLE": "Hokkaido Schnee Festival",
         "UI_PEACOCK_SOUTHERN_COMFORT": "Das Weihnachts Unglück",
         "UI_PEACOCK_SOUTHERN_COMFORT1": "Das Weihnachts Unglück - Stufe 1",
         "UI_PEACOCK_SOUTHERN_COMFORT2": "Das Weihnachts Unglück - Stufe 2",
         "UI_PEACOCK_SOUTHERN_COMFORT3": "Das Weihnachts Unglück - Stufe 3",
-        "UI_PEACOCK_SOUTHERN_COMFORT_OBJ1": "Eliminate at least 5 of the possible targets outlined in-game by pacifying them or killing them with a Christmas-themed weapon.",
-        "UI_PEACOCK_SOUTHERN_COMFORT_SANTA_TITLE": "Become Santa",
-        "UI_PEACOCK_SOUTHERN_COMFORT_SANTA_DESC": "Disguise yourself as Santa 47, do not change disguises after.",
-        "UI_PEACOCK_SOUTHERN_COMFORT_SANTA_HUD": "Become Santa, stay as Santa",
-        "UI_PEACOCK_SOUTHERN_COMFORT_KILL_SANTA": "Eliminate Santa",
+        "UI_PEACOCK_SOUTHERN_COMFORT_OBJ1": "Eliminiere mindestens 5 markierte Ziele indem du sie mit einer Weihnachtsthema Waffe tötest oder ruhigstellst.",
```

Comment:
"Weihnachtsthema Waffe" doesn't make any sense in my opinion. "Weihnachtswaffe" would probably be better.